src/CurrentBookRainbow/CurrentBookRainbow.js

```diff
@@ -55,47 +57,58 @@ const CurrentBookRainbow = ({ data, setClickedBook }) => {
+              stroke="#E3D1BE"
+              strokeWidth="22"
+              strokeLinecap="round"
               strokeLinejoin="round"
             />
-            <path 
-              d="M580.911 14C574.066 16.7085 567.536 22.4861 561.946 27.2641C555.643 32.6518 547.717 39.4227 543.282 46.7652C539.139 53.6232 535.141 61.1547 533.927 69.4642C533.263 74.0125 532.54 78.5226 532.044 83.1059C531.283 90.1452 531.933 97.2587 530.031 104.141C528.507 109.651 527.431 116.203 524.949 121.241C520.723 129.816 514.778 137.946 507.613 143.752C501.708 148.537 494.357 151.103 487.663 154.321C480.386 157.82 473.054 161.187 466.352 165.816C453.896 174.42 446.035 190.736 438.905 204.206C435.023 211.541 433.332 221.641 432.231 229.964C431.105 238.476 429.952 246.824 429.481 255.48C428.613 271.449 430.205 291.519 418.769 303.252C414.63 307.499 410.892 312.099 406.632 316.222C402.097 320.613 396.656 324.076 391.749 328.008C384.547 333.779 375.994 337.564 368.339 342.56C358.026 349.29 348.851 359.885 343.332 371.442C336.931 384.844 334.74 399.315 334.887 414.471C335.026 428.806 341.197 442.637 340.525 457.014C339.696 474.734 332.422 492.582 322.824 506.561C313.636 519.942 299.714 528.048 287.9 538.41C284.056 541.782 279.739 544.246 276.746 548.605C273.463 553.387 270.476 559.566 268.369 565.127" 
-              stroke="#EB6825" 
-              strokeWidth="20" 
-              strokeLinecap="round" 
+            <path
+              className={data.data.length > 4 ? 'rainbow' : 'off'}
+              d="M1044.43 14C1037.85 17.3099 1031.87 23.6484 1026.73 28.9072C1020.93 34.837 1013.64 42.2898 1009.88 49.9995C1006.37 57.2006 1003.06 65.0595 1002.6 73.4443C1002.34 78.0337 1002.02 82.5905 1001.94 87.1997C1001.81 94.2788 1003.1 101.306 1001.82 108.33C1000.79 113.955 1000.31 120.577 998.284 125.817C994.843 134.735 989.648 143.364 983.032 149.788C977.578 155.082 970.486 158.295 964.107 162.099C957.171 166.235 950.171 170.244 943.909 175.454C932.272 185.137 925.902 202.091 920.006 216.145C916.795 223.798 916.015 234.008 915.663 242.396C915.302 250.975 914.9 259.393 915.206 268.057C915.769 284.039 919.15 303.886 908.809 316.595C905.067 321.195 901.755 326.111 897.881 330.599C893.757 335.377 888.647 339.313 884.112 343.668C877.455 350.06 869.274 354.596 862.097 360.256C852.428 367.882 844.237 379.255 839.774 391.259C834.597 405.18 833.709 419.789 835.211 434.871C836.632 449.136 844.015 462.359 844.631 476.739C845.39 494.462 839.742 512.89 831.433 527.671C823.479 541.82 810.338 551.139 799.498 562.516C795.971 566.218 791.892 569.058 789.3 573.668C786.458 578.724 784.036 585.145 782.435 590.872"
+              stroke="#3C9F88"
+              strokeWidth="22"
+              strokeLinecap="round"
```

Comment:
Thanks for taking the time to do this functionality.  I think it adds a special touch to the UI.